packages/react/src/modules/slack/hooks/useSlackChannels.ts

```diff
@@ -1,77 +1,86 @@
 import { useKnockClient } from "@knocklabs/react-core";
```

Comment:
Would it make sense to move any of these hooks to react-core? The components that are currently using them would only work for react web but the underlying data fetching and react query should work out of the box with React Native.

Reply:
yeah that's totally right. Let me do that in the next PR where I'm introducing the provider.